addon/plugins/image/nodes/image.ts

```diff
@@ -1,13 +1,15 @@
+import { ComponentLike } from '@glint/template';
 import {
   createEmberNodeSpec,
   createEmberNodeView,
   EmberNodeConfig,
 } from '@lblod/ember-rdfa-editor/utils/ember-node';
+import Image from '@lblod/ember-rdfa-editor/components/plugins/image/node';
 import { Node as PNode } from 'prosemirror-model';
 
 const emberNodeConfig: EmberNodeConfig = {
   name: 'image',
-  componentPath: 'plugins/image/node',
+  component: Image as unknown as ComponentLike,
```

Comment:
I read through [this issue](https://github.com/typed-ember/glint/issues/518) and it does sound like we are doing the right thing with using `ComponentLike<any>` but I might be misunderstanding things.

Reply:
(I also asked in the discord, but haven't received a response yet.)

Reply:
Is the `@glint/template` package only added for the ComponentLike type?

Reply:
It could be possible that the types included with ember 5.0 are compatible with the glint types?

Reply:
> Is the @glint/template package only added for the ComponentLike type?

Yes, that seemed to be the recommended way to type use-cases like that.

> It could be possible that the types included with ember 5.0 are compatible with the glint types?

Yea, possibly, I haven't tried, but those are also not available on older versions so I don't think we can use them yet?